### ISSUE TITLE
Use get_chunks in Hashtable

### ISSUE THREAD

**Issue by melissawm** [deshaw/versioned-hdf5#95]
Depends on #94 .

**Comment by magsol** [comment]
@melissawm I know you've long since left the project, but I wanted to get a feel for whether this should be closed or simply put in the backlog.

**Comment by melissawm** [comment]
@magsol Please feel free to close if this is no longer relevant, I haven't been watching the developments here to have an opinion. Thanks for the ping!
